bin/get_ecmwf.sh

```diff
@@ -99,98 +109,101 @@ fi
 export time_units="hours since ${init_date}"
 
 # Retrieve ml, sfc, pv and pt files
-$SRCDIR/download_ecmwf.sh $DATE $TIME $STEP
+# $BINDIR/download_ecmwf.sh $DATE $TIME $STEP
```

Comment:
leave out # sign

Reply:
It is not necessary to comment/uncomment this line, since mars is not run, if the grib file already exists anyway